### ISSUE TITLE
[TW#10852] When will the SPI-slave demo be released?

### ISSUE THREAD

**Issue by ouki-wang** [espressif/esp-idf#406]
When will the SPI-slave demo be released?

**Comment by long585** [comment]
I implement my own API
```
void set_spi_mode(spi_host_device_t host,spi_mode mode)
{
	spihost[host]->hw->slave.slave_mode = mode;
}
typedef enum{
	MASTER=0,
	SLAVE,
}spi_mode;

```

**Comment by FayeY** [comment]
Hi ouki-wang, SPI slave has been implemented in [46fa2cf](https://github.com/espressif/esp-idf/commit/46fa2cfb4680b027a93c07c8616a4aa6e76d06ab)
